tests/ImageSharp.Sandbox46/ImageSharp.Sandbox46.csproj

```diff
@@ -350,6 +350,40 @@
     <Analyzer Include="..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.Analyzers.dll" />
     <Analyzer Include="..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.CSharp.Analyzers.dll" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\ImageSharp.Drawing\ImageSharp.Drawing.csproj">
+      <Project>{2e33181e-6e28-4662-a801-e2e7dc206029}</Project>
+      <Name>ImageSharp.Drawing</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\src\ImageSharp.Formats.Bmp\ImageSharp.Formats.Bmp.csproj">
```

Comment:
These .csproj-s no longer exist. Why are you referencing them?
Also: we should remove the old .dll references in this project, and make sure everything works with the new netstandard project references.